src/metabase/pulse/render/js_svg.clj

```diff
@@ -37,7 +37,7 @@
 (def ^:private static-viz-context-delay
   "Delay containing a graal js context. It has the chart bundle and the above `src-api` in its environment suitable
   for creating charts."
-  ;; todo is this thread safe? Should we have a resource pool on top of this? Or create them fresh for each invocation
+  ;; TODO: we should have resource pool for this
```

Comment:
This seems to preempt deciding between different options in `execute-fn-name`? Do we even need to keep this TODO as well?

Reply:
yea I don't think it's needed anymore. removed